testonly/hammer/hammer.go

```diff
@@ -581,11 +577,13 @@ func (s *hammerState) String() string {
 		totalInvalidReqs += int(invalidReqs.Value(s.label(), string(ep)))
 		totalErrs += int(errs.Value(s.label(), string(ep)))
 	}
-	var latestRev int64 = -1
-	if smr := s.smrs.previousSMR(0); smr != nil {
-		latestRev = int64(smr.Revision)
+	var revStr string
+	if latestRev, found := s.sharedState.getLastReadRev(); found {
+		revStr = strconv.FormatUint(latestRev, 10)
+	} else {
+		revStr = "N/A"
 	}
```

Comment:
You can remove this clause by initializing `revStr := "N/A"`.

Reply:
Done.